css/main.css

```diff
@@ -0,0 +1,402 @@
+  to(#808080));
+
+
+  height: 36px;
+  position: relative;
+  top: 5.2rem;
+  left: -2%;
+  margin: 0 auto;
+  z-index: 900;
+  border-radius: 4px;
+  width: 845px;
+
+}
+
+.navigation-ul-list {
+  display: inline-block;
+  position: relative;
+  width: 100px;
+  height: 35px;
+  text-align: center;
+}
+
+
+.navigation-link:link,
+.navigation-link:visited{
+text-decoration: none;
+color: white;
+}
+
+.navigation-ul-list:not(:last-child) {
+  border-width: 0 2px 0 1px;
+  border-top-width: 0px;
+  border-right-width: 2px;
+  border-bottom-width: 0px;
+  border-left-width: 1px;
+  border-style: solid;
+  border-top-style: solid;
+  border-right-style: solid;
+  border-bottom-style: solid;
+  border-left-style: solid;
+  -o-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+  -ms-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+  -moz-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+  -webkit-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+}
+
+.navigation-ul-list:hover:not(:last-child) {
+  background: #575757;
+  background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAAkCAQAAADifZiHAAAAJ0lEQ…zE8JeJ4R8q%2Bo%2BKCIH%2FjEA1TAyMSIgJRkIQM9BYAGa0Dj%2FXpUiAAAAAAElFTkSuQmCC) repeat-x 0 0, url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA0AAAABCAQAAACvIOyMAAAAE0lEQVQIW2P8z8jAwMAIh0wICAAa0gETv9XXOgAAAABJRU5ErkJggg%3D%3D) repeat-y 100% 0, url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA0AAAABCAQAAACvIOyMAAAAEklEQVQIW2P8L8TwDwn%2Bh0MGAKnjDAIoAAlcAAAAAElFTkSuQmCC) repeat-y 0 0, rgba(0, 0, 0, .4);
+}
+
+
+.navigation-ul-list:not(:first-child) {
+  margin-left: -4px;
+}
+
+
+.navigation-ul-list:first-child {
+  border-top-left-radius: 4px;
+  border-bottom-left-radius: 4px;
+}
+
+
+.navigation-span {
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%,-50%);
+}
+
+.navigation-icon {
+  color: white;
+  font-size: 1.4rem;
+}
+
+
+.navigation-search {
+  position: relative;
+  top: -.8rem;
+  left: 1.5rem;
+  width: 100%;
+  padding: 2px 18px 1px 22px;
+  border-radius: 12px;
+  box-sizing: border-box;
+  background-color: grey rgba(255,255,255,.1);
+  -webkit-transition: color, text-shadow 0.3s, 0.3s;
+  transition: color, text-shadow 0.3s, 0.3s;
+}
+
+
+
+.navigation-search:hover {
+background: none;
+
+}
+
+.search-icon {
+  position: absolute;
+  top: -.5rem;
+  left: 2rem;
+  color: white;
+  z-index: 9000;
+}
+
+
+
+
+/* styling the section-hero */
+
+.section-hero {
+  height: 99.4vh;
+  position: relative;
+  background: #5e5e5e url(/img/hero_2x.jpg) 50% 100% no-repeat;
+  background-size: 1300px 1315px;
```

Comment:
Since background property is used in 199 this could be included in background property instead,

See [here](https://www.w3schools.com/cssref/css3_pr_background.asp) for more info on background shorthand property.

Typically written `background: bg-color bg-image position/bg-size bg-repeat bg-origin bg-clip bg-attachment initial|inherit;`